notebooks/data_table_transformer/main.py

```diff
@@ -0,0 +1,125 @@
+import os
+import herzog
+
+# Mock the notebook environment
+os.environ['WORKSPACE_NAME'] = "terra-notebook-utils-tests"
+os.environ['WORKSPACE_BUCKET'] = "gs://fc-9169fcd1-92ce-4d60-9d2d-d19fd326ff10"
+os.environ['GOOGLE_PROJECT'] = "firecloud-cgl"
+
+with herzog.Cell("markdown"):
+    """
+    ### Use  Pandas `pivot` command  to convert [AnVIL](https://anvilproject.org) table from  long to wide
+    - `index_column` long format column containing the unique keys of the wide formatted table rows
+    - `header_column` long format column containing the new column headers of the wide formatted table
+    - `value_columns` long format columns to carry over into the wide formatted table (These may be muxed with the
+       `header_column` names, e.g. "cram" -> "file_type-cram")
+
+    While this notebook is focused on the AnVIL data model, it can work with any Terra data table with appropriate
+    choices for `index_column`, `header_column`, and `value_columns`.
+    """
+
+with herzog.Cell("markdown"):
+    """
+    Install the newest version of [terra-notebook-utils](https://github.com/DataBiosphere/terra-notebook-utils)
+    and [terra-pandas](https://github.com/xbrianh/terra-pandas)
+    """
+
+with herzog.Cell("python"):
+    #%pip install --upgrade --no-cache-dir terra-notebook-utils
+    #%pip install --upgrade --no-cache-dir terra-pandas
+    pass
+
+with herzog.Cell("markdown"):
+    """
+    Import Pandas interface functions from [terra-pandas](https://github.com/xbrianh/terra-pandas)
```

Comment:
You may want to briefly explain the difference between pandas and terra-pandas around here.

Reply:
This notebook assumes some knowledge of Pandas. With it, the difference between pandas and terra-pandas should not be ambiguous. Without it there may be confusion, but I hope it is still somewhat usable, and somewhat pedagogical.

I’ll add links to [Pandas](https://pandas.pydata.org) to assist autodidacts.